src/python/pants/backend/codegen/protobuf/python/register.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+"""Generate Python sources from Protocol Buffers (Protobufs).
+
+See https://developers.google.com/protocol-buffers/.
+"""
+
+from pants.backend.codegen.protobuf.python import additional_fields
+from pants.backend.codegen.protobuf.python.rules import rules as python_rules
+from pants.backend.codegen.protobuf.target_types import ProtobufLibrary
+
+
+def rules():
+    return [*additional_fields.rules(), *python_rules()]
+
+
+def target_types():
+    return [ProtobufLibrary]
```

Comment:
This feels odd to me. This should already be declared in the protobuf backend, presumably? It must be, because otherwise those rules would fail to load.

Reply:
There is no `pants.backend.codegen.protobuf` backend. There's `pants.backend.codegen.protobuf.python`, and in some future, possibly `pants.backend.codegen.protobuf.java`. 

It doesn't make sense to have `pants.backend.codegen.protobuf` because it wouldn't do anything interesting beyond registering this target type.